### PR TITLE
Feature/photo desc

### PR DESCRIPTION
**Pull request by bennettscience**
Backend and database updates needed to implement #167.

### CODE REVIEW

**Comment by ildyria** [comment]
I did not test it but this look correct to me.
Before merging we will need the PR in Lychee-front to be merged first. :+1:

**Comment by d7415** [comment]
> I did not test it but this look correct to me.

Ditto

> Before merging we will need the PR in Lychee-front to be merged first.

Agreed (for release process, as this takes us to 3.2.11)

File: php/database/update_030211.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+$query  = Database::prepare($connection, "SELECT `key` FROM `?` WHERE `key` = 'image_overlay_type' LIMIT 1", array(LYCHEE_TABLE_SETTINGS));
+$result = Database::execute($connection, $query, 'update_030211', __LINE__);
+
+if ($result===false) Response::error('Could not get current image_overlay_type from database!');
+
+if ($result->num_rows===0) {
+
+	$query  = Database::prepare($connection, "INSERT INTO `?` (`key`, `value`) VALUES ('image_overlay_type', 'exif')", array(LYCHEE_TABLE_SETTINGS));
+	$result = Database::execute($connection, $query, 'update_030211', __LINE__);
+	$query 	= Database::prepare($connection, "INSERT INTO `?` (`key`, `value`) VALUES ('image_overlay_type_default', 'exif')", array(LYCHEE_TABLE_SETTINGS)))
```
Comment (ildyria) [inline php/database/update_030211.php]: `image_overlay_type_default` as a setting is not needed.

**Review by ildyria — CHANGES_REQUESTED** [review]
Cf comment. Aside from that, it looks good to me.

**Review by ildyria — APPROVED** [review]

**Comment by ildyria** [comment]
Merged to v3.2.11-candidate.

**Pull request closed without merging**